test-app/config/ember-cli-update.json

```diff
@@ -6,8 +6,8 @@
       "version": "4.11.0",
       "blueprints": [
         {
-          "name": "addon",
-          "outputRepo": "https://github.com/ember-cli/ember-addon-output",
+          "name": "app",
```

Comment:
Hmm, I'm not sure about this change. How exactly does ember-cli-update work with v2 addons? We still want to be able to apply the v2 addon blueprint with this config and have it affect both the `addon` and `test-app` directories.

Reply:
@elwayman02 this is actually correct change.
this file will be used only for test app so we could do `npx ember-cli-update` in the `test-app` folder.

for the addon and everything, there will be separate `config/ember-cli-update.json` file which we would be able to use to run `npx ember-cli-update` in the root folder (you could see the template for that file in https://github.com/embroider-build/addon-blueprint/blob/main/files/config/ember-cli-update.json)

Reply:
Ok, so this file is fine but we need to add a new `config/ember-cli-update.json` as well then, right?

Reply:
Yeap, but I would do in the separate PR as it would not make any sense just yet

Reply:
Yup, seems fine!